.github/workflows/integration-tests.yml

```diff
@@ -114,7 +115,6 @@ jobs:
           path: ./integration-tests/logs
   solana:
     name: Solana Tests
-    environment: integration
```

Comment:
Was this intentional? I believe we just added this recently.

Reply:
This is why I am adding the linting in the first place. This line is invalid in both places and breaks the job.